packages/obonode/obojobo-chunks-math-equation/math-equation-properties-modal.js

```diff
@@ -112,6 +159,16 @@ class MathEquationProperties extends React.Component {
 					</div>
 					<span>{this.state.error}</span>
 				</div>
+				<div id="katexContainer" style={{ display: this.latexCheck(this.state.content.latex) }}>
```

Comment:
I think we should always show element this since it causes the dialog to suddenly expand as soon as you type something into the first input.

However, I did want to give some feedback on this (as if we were gonna keep it) for instructional purposes. I feel that latexCheck isn't a descriptive name because it doesn't tell you that it's going to return a string of 'none' or 'block'. Instead, I think latexCheck should simply return a boolean, and you can call it something like `isLatexValid`. So if we were gonna keep this then this line would become

`style={{ display: this.isLatexValid(this.state.content.latex) ? 'block' : 'none'}}`

That way we have a simple method we can test, isLatexValid, which is descriptive, and the specific css-relevant values of 'block' and 'none' are kept inside the CSS we're writing.